src/debian.rs

```diff
@@ -45,6 +45,18 @@ impl Pkg {
             false
         }
     }
+
+    pub fn show_dependencies(&self) -> bool {
+        if !self.in_debian() {
+            return true;
+        }
+
+        if let Some(deb) = &self.debinfo {
+            !deb.exact_match
```

Comment:
This would potentially show a lot more packages that we don't need to care about right now when uploading a new package/upgrading one. If Debian has a semver compatible crate available we currently stop because we can usually just use that and don't need to keep traversing the tree.

Reply:
This is sort of tricky.  Crate A may make an API compatible update which introduces new dependencies.  Hiding those gives a false sense of the work needed.

However, I guess we could say the new update doesn't matter since it's API compatible, so we just stick with the older version of the crate.

This is currently an area where debstatus is lacking, since we rely on how cargo is resolving dependencies, which is greedy.  It may say we need a new version of a crate (which has the new dependency) when we really don't.

If we instead worked with Cargo.toml versions, we could provide a more accurate picture and I'd agree more with your stance.

Reply:
> If Debian has a semver compatible crate available we currently stop because we can usually just use that and don't need to keep traversing the tree.

That's actually what my "This PR" image above shows, but then I made the change to use `exact_match` instead.  I've changed the behavior now so that it only shows dependencies for a package in Debian if it's outdated/incompatible.  This can be overridden with the "-a" switch.